pytest.ini

```diff
@@ -1,11 +1,14 @@
 [pytest]
-addopts = --verbose -ra
-python_paths = src/ tests/
+addopts = --verbose -ra --runslow
+pythonpath = src/ tests/
 env =
     COGNITO_DEPLOYMENT_MODE=Development
     SECRET_KEY=changme
     DEFAULT_X_GOPHISH_CONTACT=test@test.com
     LOCAL_API_KEY=somerandomkey
     PYTEST_SETTINGS=1
+    PYTESTING=True
+    RECOMMENDATIONS_PATH=src/static/recommendations.json
+    TEMPLATES_PATH=src/static/templates.json
```

Comment:
As long as we are cleaning things up, can these config options and env variables be alphabetized?

Reply:
resolved here https://github.com/cisagov/con-pca-api/pull/632/commits/890327965f766aecb99dc75f6642a086415da1bb